frontend/src/modules/teamcomments/actions.js

```diff
@@ -47,6 +49,12 @@ export function get(entity: number, locale: string): Function {
 
         let content = await api.entity.getTeamComments(entity, locale);
 
+        // The default return value of aborted requests is {},
+        // which is incompatible with reducer
+        if (isEmpty(content)) {
```

Comment:
Is this a ride-along?

Reply:
Yes and no. It fixes the JS error when you e.g. quickly move between strings (which aborts requests), but it only occurs now that the Comments panel is visible by default.